### PR TITLE
fixes IProjectTreePropertiesProvider doc link

### PR DESCRIPTION
**Pull request by workgroupengineering**
- fix IProjectTreePropertiesProvider doc link

### CODE REVIEW

File: doc/scenario/provide_custom_icons_for_the_project_or_item_type.md [hunk 1]
```diff
@@ -29,11 +29,11 @@ In order to add custom icons to your project, you need to follow these steps:
   tree = tree.SetIcon(Images1Monikers.ProjectIconImageMoniker.ToProjectSystemType());
   ```
 
-##Adding an image manifest manually
+## Adding an image manifest manually
 1. Image manifest
   1. Create a new xml file, name it to .imagemanifest (e.g. `Images1.imagemanifest`)
   
-    ```xml
+```xml
```
Comment (drewnoakes) [inline doc/scenario/provide_custom_icons_for_the_project_or_item_type.md:36]: This indentation was intentional, as it nests the code block under the bullet point.

File: doc/scenario/provide_custom_icons_for_the_project_or_item_type.md [hunk 2]
```diff
@@ -4,13 +4,13 @@ Provide custom icons for the Project Type/Item type
...
 
-##Tutorial
+## Tutorial
 In order to add custom icons to your project, you need to follow these steps:
 
 1. Add New Item
   - **Visual Studio 2017**: `Project Tree Properties Provider extension`
   - **Visual Studio 2015**: `Project Tree Modifier extension`
-  - This will add an [IProjectTreePropertiesProvider](../extensibility/IProjectTreePropertiesProvider)/[IProjectTreeModifier](../extensibility/IProjectTreeModifier.md) export to your project type that will replace the project icon with a JavaScript Icon
+  - This will add an [IProjectTreePropertiesProvider](../extensibility/IProjectTreePropertiesProvider.md)/[IProjectTreeModifier](../extensibility/IProjectTreeModifier.md) export to your project type that will replace the project icon with a JavaScript Icon
```
Comment (drewnoakes) [inline doc/scenario/provide_custom_icons_for_the_project_or_item_type.md:13]: 👍

**Review by drewnoakes — CHANGES_REQUESTED** [review]
Please revert the formatting changes.

File: doc/scenario/provide_custom_icons_for_the_project_or_item_type.md (hunk 1 above)
Comment (workgroupengineering) [reply doc/scenario/provide_custom_icons_for_the_project_or_item_type.md]: Ok , but the problem is that the code block is not displayed correctly when there is an indent.

Comment (drewnoakes) [reply doc/scenario/provide_custom_icons_for_the_project_or_item_type.md]: I see what went wrong. The indentation within a list must be a further three spaces for the next level. I added a commit that fixes the indentation throughout the document.

**Review by drewnoakes — APPROVED** [review]

**Comment by drewnoakes** [comment]
@workgroupengineering thanks for your contribution. Would you mind signing the CLA for this contribution, as described in the comment above?

**Comment by drewnoakes** [comment]
Thanks @workgroupengineering!

**Pull request merged**